lsp-ui-doc.el

```diff
@@ -62,7 +62,7 @@
   :link '(custom-manual "(lsp-ui-doc) Top")
   :link '(info-link "(lsp-ui-doc) Customizing"))
 
-(defcustom lsp-ui-doc-enable t
```

Comment:
IMHO better this to stay but set lsp-ui-doc-show-with-cursor  to nil. Having mouse wont be noisy.

Reply:
@yyoncho but they are already `t` and even so we show with cursor hover, not only mouse